### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | 

I've removed any `function_exists` or `extension_loaded` conditions and added my own tool to mock internal functions.
With that we can remove conditional run of tests in CI and all test can be passed in local machine too.

### CODE REVIEW

File: src/Rule/Email.php [hunk 1]
```diff
@@ -15,8 +15,6 @@
 use Yiisoft\Validator\SkipOnEmptyInterface;
 use Yiisoft\Validator\ValidationContext;
 
-use function function_exists;
-
```
Comment (vjik) [inline src/Rule/Email.php:19]: Can we make this without remove `use`?

Comment (xepozz) [reply src/Rule/Email.php:19]: You can't mock internal function if it already exists

**Review by vjik — CHANGES_REQUESTED** [review]
Not using qualified functions just because of the tests is a bad solution.

**Comment by xepozz** [comment]
Hmm, looks like PHPUnit on Windows cannot load bootstrap file at startup.

**Comment by arogachev** [comment]
CI approach may be not ideal in terms of best practices, but has proved itself as a working solution with minimum source / tests code modifications. Closing for now.

**Pull request closed without merging**